src/main/scala/GuessWho/GuessWho.scala

```diff
@@ -15,29 +17,60 @@ object GuessWho extends App{
 
   private val characterList:Seq[Character] = Seq(jon, tony, bill, paul, gary, steve, jess, vicky, sarah, beff)
 
+
   private val game:GameBoard = new GameBoard(characters = characterList, defaultChosenCharacter = None)
+  private val interface: Interface = new Interface()
+
 
   private var _playersRemaining:Seq[Character] = game.getRemainingCharacters
+  private var _newTurn:Boolean = true
+  private var _winner:Boolean = false
+
+  def print_characters() = if(_newTurn){
+    interface.displayCharacters(_playersRemaining)
+    _newTurn = false
+  }
+
+  def filter_by_name(): Seq[Character] = {
+    val name: String = interface.get_name_choice()
+    val filteredPlayers = game.filterRemaining(12, name)
+    _playersRemaining = _playersRemaining.filter(x => filteredPlayers.exists(_.name == x.name))
```

Comment:
I know it doesn't seem important, but using `x` makes this quite hard to read at first